### ISSUE TITLE
a11y:  Try it Out request body params are not accessible

### ISSUE THREAD

**Issue by cjfreedman** [swagger-api/swagger-ui#9573]
### Q&A (please complete the following information)
 - OS: [e.g. macOS]  Windows 10
 - Browser: [e.g. chrome, safari]  Chrome 
 - Version: [e.g. 22]   121
 - Method of installation: [e.g. npm, dist assets]   npm
 - Swagger-UI version: [e.g. 3.10.0]   Swagger UI React 5.11.3  
 - Swagger/OpenAPI version: [e.g. Swagger 2.0, OpenAPI 3.0]  OpenAPI 3.0

### Content & configuration
Basic default configuration

### Describe the bug you're encountering
The request body params example are not accessible.

### To reproduce...
Expand a POST call to see the Request Body block.

### Expected behavior
No accessibility error :)

### Screenshots
Here's our content:
![image](https://github.com/swagger-api/swagger-ui/assets/86489380/a2971498-971e-45a6-917a-b95a3070b08d)

### Additional context or thoughts
Here's the Sort Site error and solution
![image](https://github.com/swagger-api/swagger-ui/assets/86489380/d82971f2-5e47-4959-9b75-63867a367947)
